app/src/main/java/com/srgtrujillo/quotes/quote/navigation/QuoteDetailActivityNavigator.java

```diff
@@ -0,0 +1,38 @@
+package com.srgtrujillo.quotes.quote.navigation;
+
+import android.content.Context;
+import android.content.Intent;
+
+import com.srgtrujillo.quotes.base.navigator.Navigator;
+import com.srgtrujillo.quotes.quote.detail.QuoteDetailActivity;
+import com.srgtrujillo.quotes.quote.model.Quote;
+
+public class QuoteDetailActivityNavigator implements Navigator {
+    private Context context;
```

Comment:
Add a WeakReference to better performance.

private WeakReference<Context> context;

inside constructor {
    context = new WeakReference(context);
}

context.get() to get the context.

This is a good improvement to leaks in Android improving garbage collector.